plugins/ipam/dhcp/lease.go

```diff
@@ -432,7 +433,12 @@ func backoffRetry(resendMax time.Duration, f func() (*dhcp4.Packet, error)) (*dh
 
 		log.Print(err)
 
-		sleepTime = baseDelay + jitter(time.Second)
+		if fastRetryLimit == 0 {
```

Comment:
@SilverBut I'm a bit confused here... The CNI DHCP daemon that actually acquires the lease runs in the host network namespace and isn't dependent on the container network interface at all. So I don't understand what interface "is not up" in this scenario?

Can you explain this commit a bit more? Thanks!

Reply:
Certainly! I think "is not up" is referring the whole DHCP infra, instead of the interface. DHCP is [not designed](https://datatracker.ietf.org/doc/html/rfc2131#:~:text=DHCP%20clients%20are%20responsible%20for%20all%20message%20retransmission.) for a perfect network without packet loss, and under some ~weird~ advanced network configurations, a client is not able to communicate with other hosts before an DHCP leases is granted by certain server. The DHCP server itself may also down for other reasons, including unplanned interruption or scheduled maintenance. 

So, in general, DHCP is not as reliable as other methods (like assigning address in a database like `etcd`). To make the address allocating run more smoothly, add retry here is necessary.

Reply:
Thanks for the explanation.